### DISCUSSION TITLE
"tab completion"

### DISCUSSION THREAD

**Discussion by GronbergInfo** [ohmybash/oh-my-bash#647] [General]
Hi, another question i couldnt figure out how or what i did before.

On my server i started to use omb, and there when i type say "cd dow" and tab, it completes to "cd Downloads/".
I recently installed the desktop with debian (also) and omb from scratch, and i think i did the same configs, but when i type "cd dow" and tab, it pretends it doesnt know of Downloads.

The .bashrc looks the same to me. I use brainy theme on both consoles.

Any ideas? (Im rather sure this was omb feature)

**Comment by akinomyoga** [comment]
What is the result of the following commands?

```bash
$ bind -v | grep completion-ignore-case
$ shopt -p nocaseglob
```

Are there any differences in the outputs of the above commands between server/desktop?

**Comment by GronbergInfo** [comment]
Client: 
```
1 > bind -v | grep completion-ignore-case
shopt -p nocaseglob
set completion-ignore-case on
shopt -u nocaseglob
stefan@Ares [~] ○                                                                                          [13:02:48]
1 >
```


server
```
130 > bind -v | grep completion-ignore-case
shopt -p nocaseglob
set completion-ignore-case on
shopt -u nocaseglob
[stefan@Hades10Gbe] [~] ○                                                                                  [13:03:14]
1 >
```

Now the tab completion works on the client where "cd dow" completes to "cd Downloads"

**Reply by akinomyoga** [reply]
Thanks.

> Now the tab completion works on the client where "cd dow" completes to "cd Downloads"

Is "the client" the machine for which you reported "*when i type "cd dow" and tab, it pretends it doesnt know of Downloads.*"? If so, does it mean the problem was solved or disappeared?

**Reply by GronbergInfo** [reply]
> Thanks.
> 
> > Now the tab completion works on the client where "cd dow" completes to "cd Downloads"
> 
> Is "the client" the machine for which you reported "_when i type "cd dow" and tab, it pretends it doesnt know of Downloads._"? If so, does it mean the problem was solved or disappeared?

i think it solved the problem on the client machine.

**Reply by akinomyoga** [reply]
Thanks. Then, let's close this.
